app/script_variant.py

```diff
@@ -48,10 +55,12 @@ def romanize_japanese_name_by_name_dict(word):
         return word
 
     if word in JAPANESE_NAME_DICTIONARY:
-        yomigana = JAPANESE_NAME_DICTIONARY[word]
-        return jautils.hiragana_to_romaji(yomigana)
+        yomigana_list = JAPANESE_NAME_DICTIONARY[word]
+        for x in range(len(yomigana_list)):
+            yomigana_list[x] = jautils.hiragana_to_romaji(yomigana_list[x])
+        return yomigana_list
```

Comment:
Update comment of this function to clarify that it returns an array. Don't forget to update comment whenever you change behavior of a function.